src/main/resources/db/migration/V1__imrt_ddl.sql

```diff
@@ -0,0 +1,70 @@
+/* Primary table containing all types of items and the json data from IAT */
+CREATE TABLE item (
+  key             SERIAL PRIMARY KEY, -- Primary key for items, auto-incrementing (change serial to AUTO_INCREMENT for mySQL)
+  id              VARCHAR(9)  NOT NULL, -- Item ID as specified by IAT
+  subject         VARCHAR(20) NOT NULL,
+  grade           VARCHAR     NOT NULL,
```

Comment:
For this, and all the other varchar fields, not sure if we want to specify a maximum length, or just leave them open ended for now.

Reply:
I'll ask IAT to provide max lengths of things.